components/chef-ui-library/src/molecules/chef-status-filter-group/chef-status-filter-group.scss

```diff
@@ -21,7 +21,7 @@ chef-status-filter-group {
       display: flex;
       align-items: center;
       flex-wrap: wrap;
-      min-width: 100px;
+      width: 100px;
```

Comment:
This was the only way I could figure to make the filters all the same size. There is something happening with the CSS box model that I don't completely understand. When I did not force the width to 100px, then the filter for disconnected was computed to be 113px wide, even though it had a lot of blank space on the right side

rendered element:
![Screen Shot 2019-12-18 at 2 09 58 PM](https://user-images.githubusercontent.com/37162/71127498-150ed900-21a0-11ea-97eb-117fd27d80ff.png)

box model debugger thingy:
![Screen Shot 2019-12-18 at 2 10 20 PM](https://user-images.githubusercontent.com/37162/71127526-222bc800-21a0-11ea-8b67-e3d683d3d456.png)

I'm guessing that one or more of the elements that display below the text are added to the text's width for the box sizing computation but I don't know what to do about it.

In any case, this works as long as the label is less than 100px so it's probably fine 🤷‍♀

Reply:
This shows the not-padding-padding that is still there when forced to 100px

![Screen Shot 2019-12-18 at 3 08 08 PM](https://user-images.githubusercontent.com/37162/71130982-37a4f000-21a8-11ea-9469-f7431e1c0388.png)

Reply:
This is what it looks like with the width forced to 100px change:

![Screen Shot 2019-12-19 at 9 24 43 AM](https://user-images.githubusercontent.com/37162/71194771-66bb7000-2241-11ea-9ede-dfc6ed62b938.png)